java/client/src/org/openqa/selenium/remote/ErrorHandler.java

```diff
@@ -220,7 +220,7 @@ private String duration(long duration) {
 
   private Throwable rebuildServerError(Map<String, Object> rawErrorData, int responseStatus) {
 
-    if (!rawErrorData.containsKey(CLASS) && !rawErrorData.containsKey(STACK_TRACE)) {
+    if (rawErrorData.get(CLASS) == null || rawErrorData.get(STACK_TRACE) == null) {
```

Comment:
Similar change should be applied to other parts of this function, they should check for null value instead of calling containsKey.

Reply:
Yeah, though perhaps in a separate PR?